app/components/profile-nav.js

```diff
@@ -31,4 +31,9 @@ export default Component.extend({
     return !enterprise && !!billingEndpoint;
   }),
 
+  isOrganizationAdmin: computed('model', function () {
```

Comment:
I don’t know how likely this is, but I wonder whether the dependent key should be `model.permissions.admin` instead. Presumably `model` will never change but it could be that the permissions change and get refetched somehow…? Definitely an edge case situation though. (I wouldn’t expect `isOrganization` to ever change though.)

This is likey overkill in this situation but sometimes I like to break things down to be able to make use of composed macros:

```
isOrganization: alias('model.isOrganization'),
hasAdminPermissions: alias('model.permissions.admin'),

isOrganizationAdmin: and('isOrganization', 'hasAdminPermissions'),
```

In this way, the dependent keys are just there, and the composed property will be recalculated whenever one changes. BUT I don’t think this is necessary in this case… haha hopefully this isn’t overboard-commentary 🙃